Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -10967,15 +10967,10 @@ uintAlign(_stack)
 
 uintAlign(_stack_vector)
     subp 2 * SlotSize, sc0
-    if ARM64 or ARM64E
-        loadpairq [sp], sc1, sc2
-        storepairq sc1, sc2, [sc0]
-    else
-        loadq [sp], sc1
-        loadq 8[sp], sc2
-        storeq sc1, [sc0]
-        storeq sc2, 8[sc0]
-    end
+    loadq [sp], sc1
+    storeq sc1, [sc0]
+    loadq 8[sp], sc1
+    storeq sc1, 8[sc0]
```

Comment:
I wonder if we should just have scratch vector register and use it with `loadv` / `storev`. But the future change is fine to me.